pkg/microservice/aslan/core/common/repository/models/task/jenkins.go

```diff
@@ -41,23 +42,24 @@ type JenkinsBuildParam struct {
 
 // JenkinsBuild ...
 type JenkinsBuild struct {
-	TaskType           config.TaskType     `bson:"type"                    json:"type"`
-	Enabled            bool                `bson:"enabled"                 json:"enabled"`
-	TaskStatus         config.Status       `bson:"status"                  json:"status"`
-	ServiceName        string              `bson:"service_name"            json:"service_name"`
-	Service            string              `bson:"service"                 json:"service"`
-	OnSetup            string              `bson:"setup,omitempty"         json:"setup,omitempty"`
-	Timeout            int                 `bson:"timeout"                 json:"timeout,omitempty"`
-	Error              string              `bson:"error"                   json:"error,omitempty"`
-	ResReq             setting.Request     `bson:"res_req"                 json:"res_req"`
-	ResReqSpec         setting.RequestSpec `bson:"res_req_spec"            json:"res_req_spec"`
-	JenkinsIntegration *JenkinsIntegration `bson:"-"                       json:"jenkins_integration"`
-	JenkinsBuildArgs   *JenkinsBuildArgs   `bson:"-"                       json:"jenkins_build_args"`
-	StartTime          int64               `bson:"start_time"              json:"start_time,omitempty"`
-	EndTime            int64               `bson:"end_time"                json:"end_time,omitempty"`
-	LogFile            string              `bson:"log_file"                json:"log_file"`
-	Image              string              `bson:"image,omitempty"         json:"image,omitempty"`
-	IsRestart          bool                `bson:"is_restart"              json:"is_restart"`
+	TaskType           config.TaskType             `bson:"type"                    json:"type"`
+	Enabled            bool                        `bson:"enabled"                 json:"enabled"`
+	TaskStatus         config.Status               `bson:"status"                  json:"status"`
+	ServiceName        string                      `bson:"service_name"            json:"service_name"`
+	Service            string                      `bson:"service"                 json:"service"`
+	OnSetup            string                      `bson:"setup,omitempty"         json:"setup,omitempty"`
+	Timeout            int                         `bson:"timeout"                 json:"timeout,omitempty"`
+	Error              string                      `bson:"error"                   json:"error,omitempty"`
```

Comment:
For `Timeout`, `Error`, `StartTime` and `EndTime`, should bson tag add `omitempty` to be consistent with json?